docs/POSTINSTALL-README.md

```diff
@@ -86,16 +86,16 @@ Creating a dedicated wheel user and removing wheel from your primary user helps
 > [!CAUTION]
 > If you do these steps out of order, it is possible to end up without the ability to administrate your system. You will not be able to use the [traditional GRUB-based method](https://linuxconfig.org/recover-reset-forgotten-linux-root-password) of fixing mistakes like this, either, as this will leave your system in a broken state. However, simply rolling back to an older snapshot of your system, should resolve the problem.
 
-1. ```sudo adduser admin```
-2. ```sudo usermod -aG wheel admin```
-3. ```sudo passwd admin```
+1. ```run0 adduser admin```
```

Comment:
have you tested these with run0? It's not a drop-in sudo replacement so we need to verify that it works properly for this case

Reply:
Update: I have tested this now, I've discussed the details in Discord, but in short: Every command works with `run0`, *except* for `passwd admin`.The command will only open a blank prompt, then immediately close. If you invoke the command with `sudo`, it works fine as usually expected. We could potentially explore alternatives, one suggestion @RoyalOughtness suggested is "could just `run0` for a root shell, run commands, then exit". (Or I assumed that was a suggestion 😅)

Reply:
```
run0
adduser admin
usermod -aG wheel admin
passwd admin
exit
reboot
```